autopush/scripts/drop_user.py

```diff
@@ -0,0 +1,39 @@
+import time
+
+import click
+
+from autopush.db import (
+    get_router_table,
+    Router,
+)
+from autopush.metrics import SinkMetrics
+
+
+@click.command()
+@click.option('--router_table_name', help="Name of the router table.")
+@click.option('--months-ago', default=2, help="Months ago to remove.")
+@click.option('--batch_size', default=25,
+              help="Deletes to run before pausing.")
+@click.option('--pause_time', default=1,
+              help="Seconds to pause between batches.")
+def drop_users(router_table_name, months_ago, batch_size, pause_time):
+    router_table = get_router_table(router_table_name)
+    router = Router(router_table, SinkMetrics())
+
+    click.echo("Deleting users with a last_connect %s months ago."
+               % months_ago)
+
+    count = 0
+    for deletes in router.drop_old_users(months_ago):
```

Comment:
I mainly wanted a way to rope in how frequently the pause happens, but wanted to still maximize the actual delete batch commands to the max supported by AWS. So that the approximate speed is controlled, a little slop is ok. It's entirely likely that there will be no pause at all specified since latency is probably going to reduce the max speed anyways.